### ISSUE TITLE
-Wunused-variable in arch/arm/mach-omap1/board-h2.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#1325]
via omap1_defconfig on linux-next:
```
arch/arm/mach-omap1/board-h2.c:347:34: warning: unused variable 'isp1301_gpiod_table' [-Wunused-variable]
static struct gpiod_lookup_table isp1301_gpiod_table = {
                                 ^
```

**Comment by nickdesaulniers** [comment]
Submitted: https://lore.kernel.org/lkml/20210401162032.10150-1-maciej.falkowski9@gmail.com/

**Comment by m-falkowski** [comment]
Patch accepted: https://lore.kernel.org/lkml/YKNh5ekJ1ffBZ+xd@atomide.com/

**Comment by nickdesaulniers** [comment]
fixed: 7c302314f37b44595f180198fca5ca646bce4a5f
